src/attr/_make.py

```diff
@@ -128,8 +128,12 @@ def attrib(
 
     :type validator: ``callable`` or a ``list`` of ``callable``\\ s.
 
-    :param bool repr: Include this attribute in the generated ``__repr__``
-        method.
+    :param repr: Include this attribute in the generated ``__repr__``
+        method. If ``True``, include the attribute; if ``False``, omit it. By
+        default, the built-in ``repr()`` function is used. To override how the
+        attribute value is formatted in the, pass a custom ``callable``, which
+        should take a single value, and return a string.
```

Comment:
Not sure it's worth explicitly calling out (would possibly personally lean towards "yes" slightly), and mention that the resulting string is `str`ed not `repr`ed into the final repr? I.e. if you return a string, you get no quotes, which will possibly trip some new user up.

Reply:
fixed in 6d37c8a4eaa5ffa39077ac5a64e70687b5ccde53